LIMITATIONS.md

```diff
@@ -0,0 +1,16 @@
+## Known Limitations
+
+### Java Client Limitations
+
+- Limited JDK support outside of JDK 11 and above. Unable to build libraries on JDK 21.
+
+### Glide Core Limitations
+
+- Single slot transactions:  Transaction commands must be commited to a single slot, and multiple commands that require different slots will fail. See [#199](https://github.com/valkey-io/valkey-glide/issues/199).
```

Comment:
Do we need an example there? Or for every item there?

Reply:
Let's add an inline example - what works, what doesn't  work

Reply:
It is already documented. Please close the issue.